src/main/java/net/starlark/java/eval/Eval.java

```diff
@@ -131,6 +133,7 @@ private static TokenKind execFor(StarlarkThread.Frame fr, ForStatement node)
             continue;
           case BREAK:
             // Finish loop, execute next statement after loop.
+            CoverageRecorder.getInstance().recordVirtualJump(node);
```

Comment:
I don't have any data to assess whether this causes a performance degradation even if coverage is not being recorded (which means that the instance returned will have a no-op `recordVirtualJump`). If this should turn out to be prohibitive, I could make the instance `static final` and populate it e.g. via an environment variable rather than a Bazel flag.